.github/ISSUE_TEMPLATE.md

```diff
@@ -10,11 +10,10 @@
 Include if relevant.
 
 Please execute `:let g:neomake_verbose = 3` to enable verbose output, which
-might help showing the issue already.
-Look at `:messages` to see them.
+might help showing the issue already.  Look at `:messages` for its output.
```

Comment:
> Execute `:3verb Neomake` and look at `:messages` for its output.

That limits it to the actual command being used, and some issues are about autocommands etc, so I'll leave it.